### PR TITLE
Export (Vector) Tiles - UI and readability improvements

### PR DESCRIPTION
**Pull request by JonLavi**
With these two samples, at the moment you can start an export simultaneously as many times as you want since the button doesn't disable. What this PR does is disable the button so that only one job can run at the same time.
I've also added some whitespace and extra comments to make the code a bit more readable.

### CODE REVIEW

**Review by Rachael-E — APPROVED** [review]
cool, nice improvement: much better than the one before. Still takes an age to export the thing! Might just be my internet though.

**Comment by JonLavi** [comment]
Hi @tschie , would you mind reviewing this when you have a moment? The diff makes it look a lot bigger than the change actually is..

File: tiled_layers/export-tiles/src/main/java/com/esri/samples/export_tiles/ExportTilesSample.java [hunk 1]
```diff
@@ -114,62 +115,87 @@ public void start(Stage stage) {
...
-                  alert.show();
-                }
-                Platform.runLater(() -> progressBar.setVisible(false));
-              });
-            } catch (InterruptedException | ExecutionException ex) {
-              Alert alert = new Alert(Alert.AlertType.ERROR, ex.getMessage());
-              alert.show();
-              progressBar.setVisible(false);
-              progressBar.setProgress(0);
+          try {
```
Comment (tschie) [inline tiled_layers/export-tiles/src/main/java/com/esri/samples/export_tiles/ExportTilesSample.java]: looks like an extra indent here

**Review by tschie — APPROVED** [review]

File: tiled_layers/export-tiles/src/main/java/com/esri/samples/export_tiles/ExportTilesSample.java (hunk 1 above)
Comment (JonLavi) [reply tiled_layers/export-tiles/src/main/java/com/esri/samples/export_tiles/ExportTilesSample.java]: Thanks, good spot!

**Pull request merged**